themes/vue/source/js/theme-data.js

```diff
@@ -89,5 +89,89 @@ var themeData = [
         image: 'https://raw.githubusercontent.com/creativetimofficial/public-assets/master/vue-light-bootstrap-dashboard/vue-light-bootstrap-dashboard.jpg'
       }
     ]
+  },
+  {
+    name: "MDBootstrap",
+    description: `Check out the themes, UI Kits and plugins below built by our partners from [MDBootstrap](https://mdbootstrap.com/?utm_ref_id=82665). Learn how to use Vue in professional projects along such technologies as Bootstrap. Templates & ready components make your development faster and more efficient`,
+    seeMoreUrl: "https://mdbootstrap.com/docs/vue?utm_ref_id=82665",
+    products: [
+      {
+        link: "https://mdbootstrap.com/products/vue-ui-kit?utm_ref_id=82665",
```

Comment:
The key should be `url` and not `link`.

Reply:
Currently links are not working because of this: https://deploy-preview-2377--vuejs.netlify.com/resources/themes.html#MDBootstrap